src/PrestaShopBundle/Controller/Admin/Improve/Design/ThemeController.php

```diff
@@ -56,6 +57,7 @@
 use PrestaShopBundle\Security\Annotation\AdminSecurity;
 use PrestaShopBundle\Security\Annotation\DemoRestricted;
 use PrestaShopBundle\Security\Voter\PageVoter;
+use Symfony\Component\Debug\Exception\OutOfMemoryException;
```

Comment:
I see `OutOfMemoryException` belongs to `Debug` namespace 🤔
Does this mean in production environment there's not OutOfMemoryException but either a Fatal Error or a generic Exception (so our code would not work in production env) ?

Reply:
Right - on the other hand I think this exception never should be catched - it tells exactly that the problem is something within the system itself. I'll remove its usage